gratipay/billing/exchanges.py

```diff
@@ -82,6 +83,7 @@ def create_card_hold(db, participant, amount):
             'options': { 'submit_for_settlement': False },
             'custom_fields': {'participant_id': participant.id}
         })
+        ref = result.transaction.id
```

Comment:
@whit537 I believe that it should be null if it fails in the `try...except` block but it may still have a reference in Braintree if fails on there side after the `card_hold` has been created.